CMakeLists.txt

```
@@ -333,6 +334,34 @@ endif()
 
 # TODO: Check if ADIOS2 is parallel when openPMD_HAVE_MPI is ON
 
+# External library: libcxxwrap-julia
+if(openPMD_USE_JULIA STREQUAL AUTO)
+    find_package(JlCxx 0.8.3)
+    if(JlCxx_FOUND)
+        set(openPMD_HAVE_JlCxx TRUE)
+    else()
+        set(openPMD_HAVE_JlCxx FALSE)
+    endif()
+    if(JULIA_FOUND)
+        set(openPMD_HAVE_JULIA TRUE)
+    else()
+        set(openPMD_HAVE_JULIA FALSE)
+    endif()
+elseif(openPMD_USE_JULIA)
```

As discussed: let's not add AUTO logic for Julia
```suggestion
if(openPMD_USE_JULIA)
```

---

Done.